Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -49,6 +55,10 @@ enum BTPayPalError: Error, CustomNSError, LocalizedError {
             return 6
         case .failedToCreateNonce:
             return 7
+        case .webSessionFailedToLaunch:
```

Comment:
We can't use `Int` as the raw value of the enum here since some of the enum cases have associated values and that can't be represented in Objective-C. So we need to provide it in the variable instead. We've used the raw `Int` type for error enums that don't have associated values.